src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.lib.exception.DataProcessingException;
+import mate.academy.model.Movie;
+
+public interface MovieDao {
+    Movie save(Movie movie) throws DataProcessingException;
```

Comment:
exception should be unchecked
if you extends from RuntimeException, you don't need this signature

Reply:
done